tests/test_plsdo.py

```diff
@@ -0,0 +1,35 @@
+import json
+from pathlib import Path
+from unittest import mock
+from unittest.mock import patch
+
+import pytest
+from tests.test_utils import TESTS_DIR
+
+from averell.readers.plsdo import CommentedTreeBuilder
+from averell.readers.plsdo import get_features
+from averell.readers.plsdo import parse_xml
+
+
+@pytest.fixture
+def plsdo():
+    return json.loads(Path("tests/fixtures/plsdo.json").read_text())
+
+
+def test_parse_xml(plsdo):
+    poem = parse_xml("tests/fixtures/input_plsdo.xml")
```

Comment:
Try to avoid `tests/fixtures/disco.json` and use a proper Path() instead for win/unix compatibility

Reply:
Done